python/pyspark/mllib/clustering.py

```diff
@@ -166,12 +166,39 @@ class GaussianMixtureModel(object):
     True
     >>> labels[3]==labels[4]
     True
+    >>> clusterdata_3 = sc.parallelize([[-5.1971], [-2.5359], [-3.8220],
```

Comment:
Can you just reshape clusterdata_2 instead of copying it here (to shorten this)?